DOCS/man/stats.rst

```diff
@@ -85,6 +85,18 @@ Configurable Options
     respective duration. This can result in overlapping text when multiple
     scripts decide to print text at the same time.
 
+``term_width_limit``
+    Default: -1
```

Comment:
I don't think it is correct? Looking at
```
    if (w == -1 || h == -1)
        return M_PROPERTY_UNAVAILABLE;
```

Reply:
This is an option for stats, not the propery.